runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -252,7 +251,7 @@ bool handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe)
    uint8_t interruptReason = compInfoPT->compilationShouldBeInterrupted();
    if (interruptReason)
       {
-      if (response != MessageType::compilationCode)
+      if (response != MessageType::compilationEndCode)
```

Comment:
Here we need both `compilationEndCode` and `compilationEndErrCode` because if the compilation finished with an error code it doesn't make sense to send another message to the server to request to interrupt the compilation.